lfsapi/endpoint_finder.go

```diff
@@ -188,8 +188,17 @@ func (e *endpointGitFinder) NewEndpoint(rawurl string) Endpoint {
 	case "":
 		return endpointFromBareSshUrl(u.String())
 	default:
-		// Just passthrough to preserve
-		return Endpoint{Url: rawurl}
+		if strings.HasPrefix(rawurl, u.Scheme+"::") {
+			// Looks like a remote helper; just pass it through.
+			return Endpoint{Url: rawurl}
+		} else {
```

Comment:
Nit: we can drop this `} else {` and de-dent the below lines, since the `if` block returns unconditionally.

Reply:
You're right.  I'll fix that.